css/viewer_title.css

```diff
@@ -0,0 +1,29 @@
+.title-container {
+  font-size: 20px;
+  font-weight: 400;
+  margin-top: 20px;
+  display: flex;
+  justify-content: center;
+}
+
+.title {
+  color: #d8d8d8;
+  display: flex;
+}
+
+.info-button {
+  padding-left: 7px;
+}
+
+.info-button svg {
+  background-color: black;
+  color: white;
+}
+
+.info-button svg:hover {
+  color: #E7E4F2;
+}
+
+.info-button svg:active {
+  color: #B59FF6;
+}
```

Comment:
Would be nice to define these colors in `constants.css`!

Reply:
Yes, there is an open [PR](https://github.com/simularium/nbsv/pull/8) that defines `colors.css`, just waiting on eyes from Megan and we'll get those converted.